prow/cmd/plank/main.go

```diff
@@ -128,8 +131,20 @@ func main() {
 		logrus.WithError(err).Fatal("Error creating plank controller.")
 	}
 
+	pjClientset, err := o.kubernetes.ProwJobClientset(cfg().PodNamespace, o.dryRun)
```

Comment:
We need the `ProwJobNamespace`, not the Pod one :)

Reply:
Updated.